python/kserve/kserve/storage/storage.py

```diff
@@ -136,12 +136,21 @@ def _update_with_storage_spec():
 
     @staticmethod
     def get_S3_config():
+        # default s3 config
+        c = Config()
+
         # anon environment variable defined in s3_secret.go
         anon = ("True" == os.getenv("awsAnonymousCredential", "false").capitalize())
+        # S3UseVirtualBucket environment variable defined in s3_secret.go
+        # use virtual hosted-style URLs if enabled
+        virtual = ("True" == os.getenv("S3_USER_VIRTUAL_BUCKET", "false").capitalize())
```

Comment:
We can use lower() to do case insensitive checks, with the above check if someone sets to TRUE then it won’t work.

Reply:
done